src/manager/queue.manager.ts

```diff
@@ -48,4 +51,32 @@ export class QueueManager {
 	public deleteQueues() {
 		return this.queueDatastore.removeAll();
 	}
+	
+	public async getManyByOptions(currentPage: number, itemsPerPage: number, sortBy?: string, sortOrder?: number): Promise<any> {
```

Comment:
the name "getManyByOptions" is generic, use something more suggestive like getQueuesPaginated